media/img/icons/connection.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38.41 41.33" width="38" height="41">
```

Comment:
White version of Connection icon. I filed an issue on https://github.com/mozilla/protocol-assets/issues/49 to add VPN assets there for eventual Protocol consumption.